pkg/cri/resource-manager/policy/builtin/topology-aware/pod-preferences.go

```diff
@@ -343,20 +343,20 @@ func checkReservedPoolNamespaces(namespace string) bool {
 	return false
 }
 
-func checkReservedCPUsAnnotations(c cache.Container) bool {
+func checkReservedCPUsAnnotations(c cache.Container) (bool, bool) {
```

Comment:
How are the policy changes related??

Reply:
Argh... not in any way. That was supposed to be a separate PR... I just hacked it on the same branch and failed to drop the patch afterwards. Thanks for noticing.